src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -605,6 +609,9 @@ def get_default_realm(self):
                 return realm_info['name']
         return None
 
+    def get_default_zonegroup(self):
```

Comment:
i didn't do this since we have above the `get_default_realm` function which is also doing a similar thing. So to match with that, I omitted the `_name` in the method call.